airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1031,27 +1036,6 @@ class Deals(CRMSearchStream):
     last_modified_field = "hs_lastmodifieddate"
     associations = ["contacts", "companies"]
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        self._stage_history = DealStageHistoryStream(**kwargs)
```

Comment:
I think we need `DealStageHistoryStream` class from code, not only where it is used. It seems `DealStageHistoryStream` class was used only in `Deals` class.

Reply:
Removed in [8db98f2](https://github.com/airbytehq/airbyte/pull/10707/commits/8db98f2492064fa94868c42af2a59ae0ec977e8d).